lib/questions.rb

```diff
@@ -1,61 +1,73 @@
 # keep only the elements that start with an a
 def select_elements_starting_with_a(array)
+  array.select{|element| element[0]=='a'}
```

Comment:
Surrounding space missing for operator '=='.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.